### DISCUSSION TITLE
Matrix room

### DISCUSSION THREAD

**Discussion by GaetanLepage** [nix-community/nixvim#112] [General]
Hi @pta2002 !

What do you think about creating a matrix room to chat about nixvim development ?
It could make it easier to quickly discuss ideas without spamming GitHub.

I can create it if you wish :)

**Comment by pta2002** [comment]
Hi, I had initially said no to this but since there's clearly some sort of demand, so I've created [#nixvim:matrix.org](https://matrix.to/#/#nixvim:matrix.org). I'm @pta02 there because I already had @pta2002 but forgot my login 😅. I'll update the README to reflect this.

**Reply by GaetanLepage** [reply]
Great !

**Comment by GaetanLepage** [comment]
@Alexnortung maybe you will want to join :)
